presentation/src/main/java/com/sopt/presentation/calendar/CalendarViewModel.kt

```diff
@@ -0,0 +1,14 @@
+package com.sopt.presentation.calendar.viewmodel
+
+import com.sopt.core.util.BaseViewModel
+import com.sopt.presentation.calendar.CalendarSideEffect
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class CalendarViewModel @Inject constructor() : BaseViewModel<CalendarSideEffect>() {
+
+    fun navigateToCalendarInfoScreen() {
+        emitSideEffect(CalendarSideEffect.NavigateToInfo)
+    }
```

Comment:
P1: 화면 이름과 동일하게 navigate 함수 작성해주세요
```suggestion
    fun navigateToAppointmentCreateInfo() {
        emitSideEffect(CalendarSideEffect.NavigateToAppointmentCreateInfo)
    }
```

Reply:
요기 방 뺄게요 !!!